_content/libraries.md

```diff
@@ -1,26 +1,57 @@
-## SPSP
+## Connectors
 
-Build Interledger payments into your apps with the [ILP SPSP Client][1]. This library can connect to any SPSP receiver for account discovery and payment detail negotation. For more information on how to use SPSP, read ["SPSP: Simple Payment Setup Protocol"][2] on the Interledger Blog. 
+### Java Connector
+An Interledger [Connector implemented in Java][1]. This project is actively maintained by [Xpring](https://xpring.io) 
+and recommended for new production deployments. 
 
-## Koa ILP
+### Rust Connector
+An Interledger [Connector implemented in Rust][2]. This project is not actively maintained, but has a robust feature-set
+and is easy to use.
 
-If you use the [Koa framework][3] in Node.js, this library will allow you to monetize your API in only a few lines of code. For more information, [check out our Github][4] and [our blog post about it.][5]
+### Javascript Connector
+An Interledger [Connector implemented in Javascript][3]. This project is not actively maintained, but has been 
+battle-tested in various production deployments.
 
-## ILP Fetch
+## Libraries
```

Comment:
"Interledger primitives" seems a little vague. Could we clarify better what that means, e.g. STREAM sender + recipient to send & receive payments, functionality to peer/connect to other nodes?

Reply:
Not opposed, but was trying to keep this short. In general, I think each implementation should clearly advertise what it supports (e.g., STREAM or whatever).

How about: "Build ILP applications natively in Java using [Quilt][4], which is a Java implementation of all Interledger."

Reply:
Minor edit, maybe frame it in terms of what I as a dev can do with this?

> "Build ILP applications that send and receive payments natively in Java using [Quilt][4], which is a Java implementation of Interledger."

Reply:
Agree - I made that change to all 3. Take another look?